Formula/grepip.rb

```diff
@@ -0,0 +1,19 @@
+class Grepip < Formula
+  desc "Filters IPv4 & IPv6 addresses with a grep-compatible interface"
+  homepage "https://ipinfo.io"
+  url "https://github.com/ipinfo/cli/archive/grepip-1.0.1.tar.gz"
+  sha256 "5e03a5ea3ce631daa4d616809a035c440c11ef5cb69bbd161e067d56c6dcb402"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "./grepip/build.sh"
+    bin.install "build/grepip"
+  end
+
+  test do
+    assert_equal version.to_s, shell_output("#{bin}/grepip --version").chomp
+    assert_equal "1.1.1.1\n", `echo "asdf 1.1.1.1 asdf" | #{bin}/grepip -o`
```

Comment:
```suggestion
    assert_equal "1.1.1.1", pipe_output("#{bin}/grepip -o", "asdf 1.1.1.1 asdf").chomp
```

Reply:
This fails because grep-like tools expect an input source as the args, not the actual input content itself. So the content `asdf 1.1.1.1 asdf` has to be coming from stdin or be in a file passed as an arg.

Reply:
To be clear that 2nd arg which is the "input" is passed as a command-line arg, right? Or is this actually piped in? (It failed locally for me so I'm assuming arg, but can't tell for sure).

Reply:
I think I'll just copy the style of tests from grep.rb and grepcidr.rb; these write to and grep from files.

Reply:
It's piped in: https://rubydoc.brew.sh/Homebrew/Assertions.html#pipe_output-instance_method

Reply:
> It failed locally for me

Did `grepip` return a non-zero return status? If so, you need to give `pipe_output` a _third_ argument (see the link I shared previously).

Reply:
Actually this revealed a bug; it wasn't properly grepping on last lines that don't end with a newline char. Will fix that and bump the version.

Reply:
All tests pass now, including the one suggested here:

```bash
$ brew test grepip
==> Testing grepip
==> ~/.linuxbrew/Cellar/grepip/1.0.2/bin/grepip --version
==> ~/.linuxbrew/Cellar/grepip/1.0.2/bin/grepip -o
==> ~/.linuxbrew/Cellar/grepip/1.0.2/bin/grepip --exclude-reserved access.log
```